pipelines/resource_type_test.go

```diff
@@ -131,6 +131,26 @@ var _ = Describe("Configuring a resource type in a pipeline config", func() {
 		})
 	})
 
+	Context("with custom resource types that have params", func() {
+		BeforeEach(func() {
+			flyHelper.ConfigurePipeline(
+				pipelineName,
+				"-c", "fixtures/resource-types-with-params.yml",
+				"-v", "origin-git-server="+originGitServer.URI(),
+				"-v", "s3_bucket=jghiloni-psrtest",
```

Comment:
this is fine for now; once i re-create the bucket with the script you provide, i'll update it during the merge